tests/Behat/Page/Admin/ShowPage.php

```diff
@@ -20,10 +37,64 @@ public function getRouteName(): string
         return 'sylius_invoicing_plugin_admin_invoice_show';
     }
 
+    public function hasBillingData(
+        string $customerName,
+        string $street,
+        string $postcode,
+        string $city,
+        string $countryName
+    ): bool {
+        $billingDataText = $this->getElement('billing_address')->getText();
+
+        return
```

Comment:
WDYT about 
```
stripos($billingDataText, $customerName) && stripos($billingDataText, $street) !== false && ...
```

?

Reply:
IMO it doesn't change anything, so I let it as it is